### ISSUE TITLE
Disable output of Kirki styles - solution

### ISSUE THREAD

**Issue by LebCit** [themeum/kirki#1335]
Hello @aristath,
First, I want to congratulate you on the genius touch of `'transport' => 'auto'` !
### Issue description:
Using `'disable_output' => true` was rendering a `my-theme_no-kirki-css` stylsheet who still was rendering a `my-theme_no-kirki-inline-css`.
This was affecting the theme original styles, and outputting a weird behavior... 
### Version used:
I'm actually using Version 2.3.7

### Using theme_mods or options?
`option_type'   => 'theme_mod'`

### Quick fix
In `functions.php`
```
/**
 * Disable output of kirki styles if the plugin is disabled or removed.
 */
if( ! class_exists( 'Kirki' ) ) {
    function my_theme_remove_kirki_styles() {
        wp_dequeue_style( 'my-theme_no-kirki' );
        wp_deregister_style( 'my-theme_no-kirki' );
    }
    add_action( 'wp_enqueue_scripts', 'my_theme_remove_kirki_styles', 20 );
}
```
Hope this can help others.
Thanks a lot for your time and all of your efforts.

SYA :-)

**Comment by aristath** [comment]
> First, I want to congratulate you on the genius touch of 'transport' => 'auto' !

Thank you, I really appreciate that! Version 3.0 will be a whole lot better, this is getting a complete rewrite. 😉 

As for the extra stylesheet that was getting added, this has already been fixed in version 3.0 so I'll go ahead and close this one 👍

**Comment by LebCit** [comment]
Can't wait to see V 3.0. Actually using Kirki for a theme that will be soon released on wp.org repo, so I have to use and recommend the repo plugin.
Thanks again 👍
